### ISSUE TITLE
Add documention for rate limiting TSM compaction parameters

### ISSUE THREAD

**Issue by zachgoldstein** [influxdata/docs.influxdata.com#1654]
Related: https://github.com/influxdata/influxdb/issues/9938
```
+  # CompactThroughput is the rate limit in bytes per second that we
+  # will allow TSM compactions to happen. Note that short bursts are allowed
+  # to happen at a possibly larger value, set by CompactBurstThroughput
+  # compact-throughput = "48m"
+
+  # CompactBurstThroughput is the rate limit in bytes per second that we
+  # will allow TSM compactions to happen.
+  # compact-burst-throughput = "48m"
```

**Comment by stevebang** [comment]
Added in `influxdb-1.6-config-fixes` branch, which includes reorganization, addition of missing settings, and review of all config settings. When complete, the PR will fix this issue.

**Comment by timhallinflux** [comment]
Looks like this was merged for 1.7.  @stevebang Let's make sure this content lands in the 1.7 branch.

**Comment by stevebang** [comment]
Added to the `influxdb-1.7` branch. Includes all OSS reorg, missing settings, and cleanup.
